admin/autoupdater/files/usr/sbin/autoupdater

```diff
@@ -252,8 +257,10 @@ local function autoupdate(mirror)
   end
 
   if not autoupdater_version.newer_than(manifest.version, old_version) then
```

Comment:
Just change the condition to `if not superforce and not autoupdater_version.newer_than(manifest.version, old_version) then` (and you might want to adjust the messages to be less confusing then). Also, the variable should not be called `superforce`, but match whatever option name we choose.